ppl/src/main/antlr/OpenDistroPPLParser.g4

```diff
@@ -283,32 +300,16 @@ valueList
     : LT_PRTHS literalValue (COMMA literalValue)* RT_PRTHS
     ;
 
-fullColumnName
-    : simpleId DOT_ID*
-    ;
-
 constant
```

Comment:
what is the difference between constant and literal

Reply:
The constant includes literal, and also includes negative constants and Boolean constants. It is used as an argument in functions. We could say the constant is more like used in the mathematical situation